jetty-ee10/jetty-ee10-webapp/src/main/java/org/eclipse/jetty/ee10/webapp/WebInfConfiguration.java

```diff
@@ -207,17 +207,14 @@ public void unpack(WebAppContext context) throws IOException
                 // Look for sibling directory.
                 Path extractedWebAppDir = null;
 
-                if (war != null)
-                {
-                    Path warPath = context.getResourceFactory().newResource(war).getPath();
+                Path warPath = webApp.getPath();
```

Comment:
This will return null if `webApp` is a `CombinedResource`.   I don't think it matters to  the code below (as a combined wont be a war), but it would be good to write the code in a way to avoid getting a path from a resource unless we know it is a war.